programs/local/LocalServer.cpp

```diff
@@ -724,6 +721,7 @@ void LocalServer::addOptions(OptionsDescription & options_description)
         ("structure,S", po::value<std::string>(), "structure of the initial table (list of column and type names)")
         ("file,f", po::value<std::string>(), "path to file with data of the initial table (stdin if not specified)")
 
+        ("format,f", po::value<std::string>(), "default input and output format")
```

Comment:
How is it supposed to work with `"input-format"` and `"output-format"` parameters?